.github/workflows/test.yml

```diff
@@ -132,6 +132,7 @@ jobs:
         uses: nick-fields/retry@v2
         with:
           max_attempts: 3
+          timeout_seconds: 900
```

Comment:
note- `timeout_minutes: 15`  would also work here, it just requires one or the other